doc/en/reference/reference.rst

```diff
@@ -2070,7 +2066,26 @@ passed multiple times. The expected format is ``name=value``. For example::
     "auto" can be used to explicitly use the global verbosity level.
 
 
-.. confval:: xfail_strict
+.. confval:: strict
+
+    If set to ``True``, enables all strictness options: :confval:`strict_config`, :confval:`strict_markers`,
+    :confval:`strict_xfail`, and :confval:`strict_parametrization_ids`.
```

Comment:
I think a list reads better:

```suggestion
    If set to ``True``, enables all strictness options: 
    
    * :confval:`strict_config`
    * :confval:`strict_markers`
    * :confval:`strict_parametrization_ids`
    * :confval:`strict_xfail`
```

Reply:
Changed. I also added a note that plugins may also set their own strictness options when `strict` is enabled. At least I hope they adopt it :)